bayes_optim/base.py

```diff
@@ -544,7 +548,7 @@ def tell(self, X, func_vals, warm_start=False):
 
             self._logger.info(
                 '#{} - fitness: {}, solution: {}'.format(
-                    i + 1, func_vals[i], X_[i]
+                    i + 1, func_vals[i], X[i].to_dict
```

Comment:
I was thinking to always show the solution in the format used in the evaluation function. Perhaps you had a different opinion?